README.md

```diff
@@ -102,7 +102,7 @@ or use it with rust, like so:
 
 > $ RUSTC_WRAPPER=[path to sccache] cargo build
 
-Sccache (tries to) support gcc, clang and MSVC. If you don't [specify otherwise](#storage-options), sccache will use a local disk cache.
+Sccache (tries to) support gcc, clang, diab and MSVC. If you don't [specify otherwise](#storage-options), sccache will use a local disk cache.
```

Comment:
I think making diab a link to the diab compiler site would be useful, since it's not as well-known as the others. (I personally had never heard of it.)

Reply:
Lucky you!

I'll add a link...

Reply:
I added link.